src/desktop.ts

```diff
@@ -1,10 +1,11 @@
 import { resolveContainer, registerContainer, ContainerRegistration, container } from "./registry";
-import * as Container from "./container";
+import { Container } from "./container";
 import * as Default from "./Default/default";
 import * as Electron from "./Electron/electron";
 import * as OpenFin from "./OpenFin/openfin";
 
 const exportDesktopJS = {
+    Container,
     container,
```

Comment:
the lowercase container is the "singleton" container instance created via a resolveContainer() invocation. This might be confusing with having static access via desktopjs.Container now too. The lowercase container singleton actually never seemed to work so I'll probably rename or drop it.  resolveContainer() caches the created instance unless you call with optional force of true so it replaces a need for having a singleton to be exported.

Reply:
I'm in favor of removing

Reply:
Will create an issue for later separate PR

Reply:
https://github.com/Morgan-Stanley/desktopJS/issues/59